Source/WebKit/Platform/IPC/Decoder.h

```diff
@@ -181,8 +181,8 @@ class Decoder {
 #if PLATFORM(MAC)
     ImportanceAssertion m_importanceAssertion;
 #endif
-#if PLATFORM(COCOA)
-    HashSet<ClassStructPtr> m_allowedClasses;
+#ifdef __OBJC__
+    AllowedClassHashSet m_allowedClasses;
 #endif
```

Comment:
Welp, this was the cause of all the layout test crashes.

Turns out that only defining an instance variable in Objective-C++ sources (but not C++ sources) results in a "bad time" when an object is passed between code in those two source types.

New PR coming soon.